plugins/modelinspector/modelcontentproxymodel.h

```diff
@@ -50,7 +50,8 @@ class ModelContentProxyModel : public QIdentityProxyModel
 public:
     enum Roles {
         DisabledRole = GammaRay::UserRole + 1,
-        SelectedRole
+        SelectedRole,
+        EmptyNameRole
```

Comment:
I'd name that a bit more descriptive, e.g. `IsDisplayStringEmptyRole`

=> You'll know it checks the display role string
=> You'll know it'll return a bool